hknweb/events/templates/events/index.html

```diff
@@ -93,5 +93,6 @@ <h3  {% if messages.tags %} class="{{ message.tags }}" {% endif %} style="color:
         <button type='submit'>Add an event</button>
     </form>
     {% endif %}
+    <br>
```

Comment:
What does this <br> do? Just spacing at the bottom?

I'd personally be adjusting the padding / margin at the bottom of the content instead of adding a paragraph break here.

Reply:
Unfortunately we've already been doing this in a lot of other places... Do you feel strongly about this? Besides css sucks